mailer.rst

```diff
@@ -1483,6 +1483,20 @@ the email is sent::
         // do something with the message
     }
 
+.. versionadded:: 6.3
```

Comment:
Mostly, not for new components, we move the versionadded below the docs

Reply:
Moved